### PR TITLE
update learning pathways for seurat and scanpy

### PR DESCRIPTION
**Pull request by nomadscientist**
This way both learning paths point to both packages

### CODE REVIEW

**Comment by nomadscientist** [comment]
I can't figure out the latest build failure

File: learning-pathways/reloaded_single_cell.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        topic: galaxy-interface
+
+  - section: "Module 2: Preparing the dataset"
+    description: |
+      These tutorials take you from raw scRNA sequencing reads to a matrix ready for downstream analysis. Galaxy coding environments don't have the same level of computational power as the easy-to-use Galaxy tools, so in practice, dataset preparation is usually performed in the Galaxy user interface to process the dataset into something smaller, which can then be analysed in the coding environment. Nevertheless, the whole process can be performed in a coding environment.
+    tutorials:
+      - name: alevin-commandline
+        topic: single-cell
+
+  - section: "Module 2: Generating cluster plots"
```
Comment (MarisaJL) [inline learning-pathways/reloaded_single_cell.md:41]:
```suggestion
  - section: "Module 3: Generating cluster plots"
```

File: learning-pathways/reloaded_single_cell.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    description: |
+              These tutorials take you from the pre-processed matrix to cluster plots and gene expression values. You can pick whether to follow the Python (Scanpy) or R (Seurat) tutorial.
+    tutorials:
+      - name: scrna-case-jupyter_basic-pipeline
+        topic: single-cell
+      - name: scrna-case_FilterPlotandExploreRStudio
+        topic: single-cell
+
+  - section: "Module 3: Inferring trajectories"
```
Comment (MarisaJL) [inline learning-pathways/reloaded_single_cell.md:50]:
```suggestion
  - section: "Module 4: Inferring trajectories"
```

**Review by MarisaJL — COMMENTED** [review]
I think you just need to change these numbers!

**Pull request merged**